internal/providers/azure/objectstore.go

```diff
@@ -164,6 +168,12 @@ func (s *ObjectStore) getLogger() logrus.FieldLogger {
 	})
 }
 
+func isValidStorageAccount(storageAccount string) bool {
```

Comment:
reaname it to isValidStorageAccountName, or simply isValid and add godoc explayning what "valid" means eventually a link to the regexp or the description of a valid storage account name

Reply:
thx, fixed